app/models/conversion_host.rb

```diff
@@ -274,16 +277,39 @@ def miq_ssh_util_args_manageiq_providers_openstack_cloudmanager_vm
   # +extra_vars+ option should be a hash of key/value pairs which, if present,
   # will be passed to the '-e' flag.
   #
-  def ansible_playbook(playbook, extra_vars = {})
-    command = "ansible-playbook #{playbook} -i #{ipaddress}"
+  def ansible_playbook(playbook, extra_vars = {}, auth_type = nil)
+    host = hostname || ipaddress
+
+    command = "ansible-playbook #{playbook} --inventory #{host}, --become -vvv"
```

Comment:
Should the comma be there after the host?

Reply:
Yes, that's how you define the inventory from a list of hostnames / IP addresses. Otherwise, it looks for a file.